src/axom/quest/IntersectionShaper.hpp

```diff
@@ -326,6 +329,9 @@ class IntersectionShaper : public Shaper
                                                                 m_octs,
                                                                 m_octcount);
 
+    // Oddities required by hip
```

Comment:
I agree, this is an oddity.  But the comment should explain why it's needed, such as:
```suggestion
    // Oddities required by hip: get a local copy to avoid capturing `this`.
```